README.rst

```diff
@@ -2,20 +2,33 @@
 ``sunpy``
 *********
 
-|Latest Version| |DOI| |matrix| |codecov| |Binder| |Powered by NumFOCUS|
+|Latest Version| |DOI| |repostatus| |python|
+|ci| |codecov| |Docs|
+|matrix|  |Binder|
+|Powered by NumFOCUS|
```

Comment:
If you wanted to make these appear on different lines you didn't succeed according to the preview.

Reply:
I didn't, just thought in the source it was a bit clearer to group them.